app/controllers/practice_origin_facilities.rb

```diff
@@ -0,0 +1,16 @@
+class PracticeOriginFacilities < ApplicationController
+  def new
```

Comment:
do we need the `new` and `create` methods?

Reply:
actually, i don't know what this file is for, lol
it's a controller with an initialize... is this even being used?